fcrepo-client-impl/src/main/java/org/fcrepo/client/utils/TransactionIdHolder.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.client.utils;
+
+/**
```

Comment:
Does this class provide functionality beyond what using a ThreadLocal<String> variable in FedoraRepositoryImpl.java?

Also, please include a class description and @since statement in the Javadoc comment.